src/components/__tests__/InfiniteHits.js

```diff
@@ -25,8 +30,10 @@ const defaultState = {
   results: {
     page: 0,
   },
+  isFirstPage: false,
   isLastPage: false,
   showMore: () => {},
+  showPrevious: () => {},
```

Comment:
since InstantSearch is a dependency here, it makes sense to only test when it's available.